draft-ietf-quic-http.md

```diff
@@ -389,6 +389,23 @@ HTTP_PUSH_ALREADY_IN_CACHE; see {{errors}}).  This asks the server not to
 transfer the data and indicates that it will be discarded upon receipt.
 
 
+## Solicited Stream Cancellation
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame to request prompt closure of the stream in the opposite
+direction.  This typically indicates that the receiving application is no longer
+reading from the stream, but does not guarantee that incoming data will be
+ignored.
+
+Upon receipt of a STOP_SENDING frame on a stream in the "open" or "half-closed
```

Comment:
I wouldn't say "on a stream," since the frame is delivered on the control channel.